pkgs/top-level/all-packages.nix

```diff
@@ -39482,7 +39482,7 @@ with pkgs;
 
   nix-melt = callPackage ../tools/nix/nix-melt { };
 
-  nixos-option = callPackage ../tools/nix/nixos-option { nix = nixVersions.nix_2_3; };
+  nixos-option = callPackage ../tools/nix/nixos-option { };
```

Comment:
@SuperSandro2000  I have a system that uses an overlay that sets `nix = prev.nixUnstable`, in which case this breaks (nixUnstable being version 2.16 at the moment).
Wouldn't it be better to set `nix = nixVersions.nix_2_15;` here instead of letting it pick up the default?

Reply:
Other tools extending nix have similar problems and just need to be updated when the default nix version is changed.